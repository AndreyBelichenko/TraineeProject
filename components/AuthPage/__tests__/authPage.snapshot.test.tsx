import * as React from 'react';
import AuthPage from '../AuthPage';
import { configure, shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from '../../../redux/configureStore';
import * as Adapter from 'enzyme-adapter-react-16';
const adapter = Adapter as any;
configure({ adapter: new adapter() });
const store = configureStore();

it('AuthPage snapshot', () => {
  const tree = shallow(<Provider store={store}><AuthPage /></Provider>);
  expect(tree).toMatchSnapshot();
});
