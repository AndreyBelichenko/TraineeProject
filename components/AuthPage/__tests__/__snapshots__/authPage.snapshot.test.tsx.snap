// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthPage snapshot 1`] = `ShallowWrapper {}`;
